app/helpers/active_admin_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ActiveAdminHelper
+  def render_translations(attribute, joiner = " ")
```

Comment:
Why this optional parameter? Wouldn't it be better to make it " | " or not optional?

Reply:
Yep! we can change the default value.